static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -375,15 +381,16 @@ $gutter-large: 55px;
 .main-navigation {
     // Unset ul from _lists.scss
     margin-left: 0;
-    // Unset ul from user agent stylesheet
-    margin-top: 0;
+    // Unset ul from user agent stylesheet / Hides the line above the first 'pillar'
```

Comment:
As the "Pillars (God, we need a different word for them)" are re-ordered using flexbox, there's no way of me targeting them using first-child or anything like that, so the best way I could think was a negative margin to make sure it's always the first item affected.